api_test.go

```diff
@@ -37,25 +37,27 @@ func (m *requestURLResolver) SetRequest(r http.Request) {
 
 type invalid string
 
-func (i invalid) GetID() string {
-	return "invalid"
+func (i invalid) GetID() jsonapi.Identifier {
+	return jsonapi.Identifier{ID: "invalid", LID: "invalid", Name: "invalid"}
```

Comment:
before this struct doesn't implement method GetName, so it's not needed to return `Name` for such cases.

Reply:
removed